src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/AbstractGenerateNuGetPackageCommand.cs

```diff
@@ -57,6 +57,8 @@ internal abstract class AbstractGenerateNuGetPackageCommand : AbstractSingleNode
             // Ensure build manager is initialized.
             await EnsureBuildManagerInitializedAsync().ConfigureAwait(true);
 
```

Comment:
Ha, it was actually a false positive, that ConfigureAwait(true) was leaving us on the UI thread.